pcstac/pcstac/filter.py

```diff
@@ -0,0 +1,90 @@
+import asyncio
+from typing import Any, Dict, List, Optional, Set
+
+import requests
+from fastapi import HTTPException, Request
+from stac_fastapi.types.core import AsyncBaseFiltersClient
+
+
+class MSPCFiltersClient(AsyncBaseFiltersClient):
+    """Defines a pattern for implementing the STAC filter extension."""
+
+    queryable_intersection = None
```

Comment:
My understanding is that once this is successfully set, it'll be cached on the object for the duration of the gunicorn process' life. Since those can be long-lived, should we instead put a TTL cache on the intersection endpoint that allows new queryables to get picked up as they're added?

Reply:
I think that's likely a more robust solution. My thinking on this was that gunicorn processes are pretty likely to die and respawn with some frequency (the exact duration is not certain a priori afaict) and this behavior is *somewhat* tuneable: https://docs.gunicorn.org/en/stable/settings.html#max-requests